nautobot/core/templates/generic/object_retrieve_plugin_full_width.html

```diff
@@ -1,3 +1,4 @@
+{% comment %}TODO - is this actually working?{% endcomment %}
```

Comment:
Template is fine, this is used by extras.api.views to render a fragment for inclusion in the page.